ts/features/cie/components/CieIdNotInstalled.tsx

```diff
@@ -36,7 +38,9 @@ const CieIdNotInstalled = ({ isUat }: CieIdNotInstalledProps) => {
               android: isUat ? CIE_ID_ANDROID_COLL_LINK : CIE_ID_ANDROID_LINK,
               default: ""
             })
-          );
+          ).catch(() => {
```

Comment:
Instead of defining a `catch` here why not using `openWebUrl` utility function? ([here](https://github.com/pagopa/io-app/blob/72cfb3ce9aedf653e6b226d7e886a4f93c6d23fe/ts/utils/url.ts#L101) the code)

Reply:
Thank you for this suggestion! Addressed [here](https://github.com/pagopa/io-app/pull/6554/commits/55487ba318bc298a7700f72e2cd3dbf9b06d9e9f)